### DISCUSSION TITLE
What difference with "include subdomains" and "httpOnly" in cookies format?

### DISCUSSION THREAD

**Discussion by sjhgithub** [curl/curl#15726] [Q&A] [answered]
I can't get the difference with "include subdomains" and "httpOnly" in cookies format,I want get help,please!
And I can't get the explanation of "include subdomains" field of cookies format?

https://github.com/curl/curl/blob/master/docs/HTTP-COOKIES.md#cookie-file-format
https://github.com/curl/curl/blob/master/docs/libcurl/opts/CURLOPT_COOKIELIST.md#example

**Comment by pkropachev** [comment]
Cookies with the HttpOnly flag are not visible to the browser code, but are sent only to the server.
Value of "include subdomains" means if cookie can be or can't be used across subdomains.

**Comment by jay** [comment, marked as answer]
https://github.com/curl/curl/blob/75a2079d5c28debb2eaa848ca9430f1fe0d7844c/lib/cookie.c#L813-L822
https://github.com/curl/curl/blob/75a2079d5c28debb2eaa848ca9430f1fe0d7844c/lib/cookie.c#L854-L861
